### DISCUSSION TITLE
Contact Algorithm  Anura3D

### DISCUSSION THREAD

**Discussion by ZeidBit** [Anura3D/Anura3D_OpenSource#20] [General]
Hey everyone, 

I am running a slope stability problem on a linear elastic base and running into an error when reading the GOM file. 

![image](https://user-images.githubusercontent.com/103214089/179807723-a4dc46c9-8435-44ed-816d-492f5a4779e6.png)

I am trying to simulate some material sliding over a linear elastic base. I put contact properties on the base between all four of the sliding material (one dry material, one liquid and two unsaturated total stress). I followed the manual for the procedure and when I run the second stage (zero quasi-static convergence) the code stops when attempting to read the contact formulation from the GOM file showing a PAUSE after the file name. 

![image](https://user-images.githubusercontent.com/103214089/179808585-7dcecb40-5f00-4f73-bbed-114a30d04c80.png)
 
Any idea why this is happening? Any help would be much appreciated! Thanks.

Zeid

**Comment by murphy-john** [comment]
Have you tried using the Debugging feature in Visual Studio? It can help you find the location where you're running into the issue.
